index.html

```diff
@@ -1,207 +1,127 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Collins Akorda - Portfolio</title>
-  <link rel="stylesheet" href="style.css">
+  <title> Collins Akorda - Portfolio</title>
+  <link rel="Stylesheet" href="Style.css">
 </head>
-
 <body>
-
+  
   <header>
     <nav>
-      <div class="nav-bar">
-        <a href="#" class="logo">
-          <img class="menuIcone" src="./images/logo.png" alt="logo Image">
-        </a>
-
-        </button-->
-        <input type="checkbox" class="side-menu" id="side-menu" />
-        <label for="side-menu" class="hamb">
-          <span class="hamb-line"></span>
-        </label>
-        <div class="nav">
-          <ul class="menu">
-            <li><a href="#Portfolio">Portfolio</a></li>
-            <li><a href="#About">About</a></li>
-            <li><a href="#contact">Contacts</a></li>
-          </ul>
-        </div>
-
+		  <div class="nav-bar">
+			    <a href="#" class="logo">Collins A.</a>
+			    <button class="hambug-menu">
+			    <i class="menuIcon material-icons"> <img class="menuIcone" src="./Images/Union.png " alt="Union Image"></i>
+          <i class="menuIcon material-icons"> <img class="menuIcone" src="./Images/icon.png " alt="Union Image"></i>
+			</button>
+			<ul class="mobile-menu">
+			  <li><a class="menuItem" href="#Portfolio">Portfolio</a></li>
+			  <li><a class="menuItem" href="#About">About</a></li>
+			  <li><a class="menuItem" href="#contact">Contacts</a></li>
+			</ul>
+	
+			<ul class="nav-links">
+			  <li class="links">Portfolio</li>
+			  <li class="links">About</li>
```

Comment:
Also as requested by the previous reviewer make sure to adjust the background color of the menu 🎨 you did a great job with the blur but the color isn't quite right yet, take a look

## What you have

![Screenshot 2023-04-21 at 20 01 50](https://user-images.githubusercontent.com/45739963/233750649-512c4d89-f0c9-463d-b5b7-093da4132e9a.png)

## What you need

![Screenshot 2023-04-21 at 20 02 15](https://user-images.githubusercontent.com/45739963/233750683-fede3150-c1f1-4691-b0c9-47766961ac6a.png)